pkg/protocol/http1/req/header.go

```diff
@@ -65,6 +66,12 @@ func WriteHeader(h *protocol.RequestHeader, w network.Writer) error {
 	return err
 }
 
+// writeTrailer writes request trailer to w
+func WriteTrailer(h *protocol.RequestHeader, w network.Writer) error {
```

Comment:
Can this be written as structure method?

Reply:
I see `fasthttp` is written as structure method in `RequestHeader`, but `hertz` do this as a param in function